examples/nirfmxwlan/ofdm-modacc-txp-composite.py

```diff
@@ -0,0 +1,513 @@
+    if response.status > 0:
+        warning_message = client.GetErrorString(
+            nirfmxwlan_types.GetErrorStringRequest(
+                instrument=instrument,
+                error_code=response.status,
+            )
+        )
+        sys.stderr.write(
+            f"{warning_message.error_description}\nWarning status: {response.status}\n"
+        )
+
+
+try:
+    initialize_response = instr_client.Initialize(
+        nirfmxinstr_types.InitializeRequest(
+            session_name=SESSION_NAME,
+            resource_name=RESOURCE,
+            option_string=OPTIONS,
+        )
+    )
+    instr = initialize_response.instrument
+
+    instr_client.CfgFrequencyReference(
+        nirfmxinstr_types.CfgFrequencyReferenceRequest(
+            instrument=instr,
+            channel_name="",
+            frequency_reference_source_mapped=nirfmxinstr_types.FREQUENCY_REFERENCE_SOURCE_ONBOARD_CLOCK,
+            frequency_reference_frequency=10e6,
+        )
+    )
+
+    client.CfgFrequency(
+        nirfmxwlan_types.CfgFrequencyRequest(
+            instrument=instr,
+            selector_string="",
+            center_frequency=2.412e9,
+        )
+    )
+
+    client.CfgReferenceLevel(
+        nirfmxwlan_types.CfgReferenceLevelRequest(
+            instrument=instr,
+            selector_string="",
+            reference_level=0.0,
+        )
+    )
+
+    client.CfgExternalAttenuation(
+        nirfmxwlan_types.CfgExternalAttenuationRequest(
+            instrument=instr,
+            selector_string="",
+            external_attenuation=0.0,
+        )
+    )
+
+    client.CfgIQPowerEdgeTrigger(
+        nirfmxwlan_types.CfgIQPowerEdgeTriggerRequest(
+            instrument=instr,
+            selector_string="",
+            iq_power_edge_source="0",
+            iq_power_edge_slope=nirfmxwlan_types.IQ_POWER_EDGE_TRIGGER_SLOPE_RISING,
+            iq_power_edge_level=-20.0,
+            trigger_delay=0.0,
+            trigger_min_quiet_time_mode=nirfmxwlan_types.TRIGGER_MINIMUM_QUIET_TIME_MODE_AUTO,
+            trigger_min_quiet_time_duration=5e-6,
+            iq_power_edge_level_type=nirfmxwlan_types.IQ_POWER_EDGE_TRIGGER_LEVEL_TYPE_RELATIVE,
+            enable_trigger=True,
+        )
+    )
+
+    client.CfgStandard(
+        nirfmxwlan_types.CfgStandardRequest(
+            instrument=instr,
+            selector_string="",
+            standard=nirfmxwlan_types.STANDARD_802_11_AG,
+        )
+    )
+
+    client.CfgChannelBandwidth(
+        nirfmxwlan_types.CfgChannelBandwidthRequest(
+            instrument=instr,
+            selector_string="",
+            channel_bandwidth=20e6,
+        )
+    )
+
+    client.SelectMeasurements(
+        nirfmxwlan_types.SelectMeasurementsRequest(
+            instrument=instr,
+            selector_string="",
+            measurements=nirfmxwlan_types.MEASUREMENT_TYPES_OFDMMODACC
+            | nirfmxwlan_types.MEASUREMENT_TYPES_TXP,
```

Comment:
```
examples/nirfmxwlan/ofdm-modacc-txp-composite.py:172: error: Argument "measurements" to "SelectMeasurementsRequest" has incompatible type "int"; expected "ValueType"  [arg-type]
```

`SelectMeasurementsRequest` contains a oneof: https://github.com/ni/grpc-device/blob/main/generated/nirfmxwlan/nirfmxwlan.proto#L3740
```
  oneof measurements_enum {
    MeasurementTypes measurements = 3;
    uint32 measurements_raw = 4;
  }
```
Protobuf enum values are ints with "newtype" annotations to avoid mixing them, but bitwise OR probably doesn't preserve the types. Try setting `measurements_raw` instead of `measurements`.

Reply:
Done. Thanks again